index.js

```diff
@@ -147,20 +147,20 @@ if (config.featurePsyPages) {
   app.get('/psychologue/logout', loginController.getLogout);
 
   //@TODO ajouter `/psychologue/`
-  app.get('/mes-seances', dashboardController.dashboard)
-  app.get('/nouvelle-seance', appointmentsController.newAppointment)
-  app.post('/creer-nouvelle-seance',
+  app.get('/psychologue/mes-seances', dashboardController.dashboard)
+  app.get('/psychologue/nouvelle-seance', appointmentsController.newAppointment)
+  app.post('/psychologue/creer-nouvelle-seance',
```

Comment:
pour les posts, je propose le prefixe "api" (on peut eventuellement mettre /psychologue/api aussi ? Comme tu veux)

```suggestion
  app.post('/api/creer-nouvelle-seance',
```